src/TransportTests/ConfigureAzureServiceBusTransportInfrastructure.cs

```diff
@@ -10,9 +10,8 @@ class ConfigureAzureServiceBusTransportInfrastructure : IConfigureTransportInfra
 {
     public TransportConfigurationResult Configure(SettingsHolder settings, TransportTransactionMode transactionMode)
     {
-        settings.Set("Transport.ConnectionString", Environment.GetEnvironmentVariable("AzureServiceBus.ConnectionString"));
-        var connectionString = settings.Get<string>("Transport.ConnectionString");
-        settings.Set<Conventions>(new Conventions());
+        var connectionString = Environment.GetEnvironmentVariable("AzureServiceBus.ConnectionString");
```

Comment:
Since setting is not longer set directly, I'd suggest to bring the line closer to where it's used, https://github.com/Particular/NServiceBus.AzureServiceBus/blob/78fde9c17b6e238764137f64491b0e2b5e4e906b/src/TransportTests/ConfigureAzureServiceBusTransportInfrastructure.cs#L31

Reply:
@SeanFeldman can you simply do it before you merge? That would be great

Reply:
👍